### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Andrey1104**
Знімок екрана (20)
Знімок екрана (21)
Знімок екрана (22)
Знімок екрана (23)
Знімок екрана (24)
Знімок екрана (25)

### CODE REVIEW

File: taxi/urls.py [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
 
 urlpatterns = [
     path("", index, name="index"),
+    path(
+        "manufacturers/",
+        ManufacturerListView.as_view(),
+        name="manufacturer-list"
+    ),
+    path("cars/", CarListView.as_view(), name="car-list"),
+    path("cars/<int:pk>", CarDetailView.as_view(), name="car-detail"),
```
Comment (Dimosphen1) [inline taxi/urls.py:20]: Add a trailing `/` to all your endpoints.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several minor changes were requested.

File: taxi/urls.py (hunk 1 above)
Comment (Andrey1104) [reply taxi/urls.py:20]: ok. can you explain why code work without slash?

Comment (Andrey1104) [reply taxi/urls.py:20]: something was wrong ))) https://github.com/mate-academy/py-taxi-service-class-based-views/pull/518 , please approve this pull request

**Pull request closed without merging**